### PR TITLE
Rename zeit.co/new → zeit.co/import

### PR DESCRIPTION
**Pull request by chibicode**
We're replacing https://zei.co/new to https://zei.co/import - I did VSCode search and replace.

[PRODUCT-1248](https://zeit.atlassian.net/browse/PRODUCT-1248)

### CODE REVIEW

File: .github/EXAMPLE_README_TEMPLATE.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This directory is a brief example of a [Name](site-link) site that can be deploy
 
 Deploy your own [Name] project with ZEIT Now.
 
-[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/zeit/now-examples/tree/master/example-directory)
+[![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/import/project?template=https://github.com/zeit/now-examples/tree/master/example-directory)
```
Comment (styfle) [inline .github/EXAMPLE_README_TEMPLATE.md:9]: I still see 404

Comment (chibicode) [reply .github/EXAMPLE_README_TEMPLATE.md:9]: Sorry @styfle, I'm on call with @leo right now and we're about to deploy and he asked me to make this ready for review

**Review by styfle — APPROVED** [review]
:shipit:

File: .github/EXAMPLE_README_TEMPLATE.md (hunk 1 above)
Comment (styfle) [reply .github/EXAMPLE_README_TEMPLATE.md:9]: Looks good now 👍

**Pull request merged**